src/main/java/com/icthh/xm/gate/gateway/ratelimiting/DiscoveryClientRouteDefinitionLocatorEnhancer.java

```diff
@@ -41,10 +44,12 @@ public Flux<RouteDefinition> getRouteDefinitions() {
 
     private RouteDefinition enrichRouteDefinition(RouteDefinition routeDefinition) {
         String routeId = clearRouteId(routeDefinition.getId()); // ReactiveCompositeDiscoveryClient adds prefix to the route IDs to avoid naming conflicts and clearly indicate their origin
-        var redisRateLimiterProperties = applicationProperties.getRedisRateLimiter().get(routeId);
+        var redisRateLimiterProperties = applicationProperties.getRedisRateLimiter();
 
         if (Objects.nonNull(redisRateLimiterProperties)) {
-            redisRateLimiterProperties.stream()
+            Optional.ofNullable(redisRateLimiterProperties.get(routeId))
+                .orElse(Collections.emptyList())
+                .stream()
                 .map(DiscoveryClientRouteDefinitionLocatorEnhancer::buildRateLimiterFilter)
                 .forEach(f -> routeDefinition.getFilters().add(f));
```

Comment:
Lets extract to separate function and will add ``log.gebug`` with information about route and filter